trigger/patch.go

```diff
@@ -362,3 +358,31 @@ func (t *patchTriggers) generate(sub *event.Subscription) (*notification.Notific
 	}
 	return notification.New(t.event.ID, sub.Trigger, &sub.Subscriber, payload)
 }
+
+func (t *patchTriggers) getGithubContext() (string, error) {
+	pRef, err := model.FindOneProjectRef(t.patch.Project)
+	if err != nil {
+		return "", errors.Wrap(err, "unable to find project ref")
+	}
+	if pRef == nil {
+		return "", errors.Errorf("project '%s' not found", t.patch.Project)
+	}
+	parentPatch, err := patch.FindOneId(t.patch.Triggers.ParentPatch)
+	if err != nil {
+		return "", errors.Wrap(err, "can't get parent patch")
+	}
+	if parentPatch == nil {
+		return "", errors.Errorf("parent patch '%s' does not exist", t.patch.Triggers.ParentPatch)
+	}
+	patchIndex, err := t.patch.GetPatchIndex(parentPatch)
+	if err != nil {
+		return "", errors.Wrap(err, "error getting child patch index")
+	}
+	var githubContext string
+	if patchIndex == 0 || patchIndex == -1 {
+		githubContext = fmt.Sprintf("evergreen/%s", pRef.Identifier)
+	} else {
+		githubContext = fmt.Sprintf("evergreen/%s/%s", pRef.Identifier, strconv.Itoa(patchIndex))
```

Comment:
I'm not sure sure what the github context is used for but, can this instead use maybe the parent patch project identifier identifier or something? I hesitate about using the index of it's location because this wouldn't be very user friendly (unless this isn't something users see, idk). 

Also, rather than using strconv.Iota, you can just use %d inside the format string directly.

Reply:
discussed over slack

Reply:
> Also, rather than using strconv.Iota, you can just use %d inside the format string directly. 
this part is still relevant though

Reply:
the change is in the commit I pushed